cmd/frontend/internal/frontend/service.go

```diff
@@ -258,25 +258,30 @@ func (fes *FrontEndService) Monitor(ctx context.Context, errCh chan<- error) {
 		return
 	}
 
-	logForced := func() bool {
-		fes.monitorMu.Lock()
-		defer fes.monitorMu.Unlock()
-		forced := fes.logNextMonitorStatus
-		fes.logNextMonitorStatus = false
-		return forced
-	}
-
-	//linkCh := make(chan string, 1)
 	go func() {
-		var once sync.Once
-		//defer close(linkCh)
 		// status of external connectivity; requires 1 GW per IP family if configured to be reachable
-		init, noConnectivity := true, true
+		hasConnectivity := false
 		connectivityMap := map[string]bool{}
 		delay := 3 * time.Second // when started grant more time to write the whole config (minimize intial link flapping)
 		_ = fes.WaitStart(ctx)
```

Comment:
The following code from the start of Monitor() should be moved after WaitStart() as well. The error _should not happen_ that case.
```
lp, err := fes.routingService.LookupCli()
	if err != nil {
		errCh <- fmt.Errorf("routing service cli not found: %v", err)
		return
	}
```

Reply:
Can WaitStart() be moved to before the go func()?

Reply:
You are right. I move them back into the go function.